@vates/task/combineEvents.js

```diff
@@ -65,6 +65,8 @@ exports.makeOnProgress = function ({ onRootTaskEnd = noop, onRootTaskStart = noo
         taskLog.end = event.timestamp
         taskLog.result = event.result
         taskLog.status = event.status
+      } else if (type === 'abortionRequested') {
```

Comment:
I think this method is starting to grow , is it possible to split  it ? 
Is the doc of the content of a task up to date ?

Reply:
1. IMHO the code is readable and I don't see an easy way to improve it :thinking: 
2. I have added a new commit which document this event, thanks :slightly_smiling_face: